test/helpers/increaseTime.js

```diff
@@ -29,7 +29,7 @@ export default function increaseTime(duration) {
 }
 
 // sets time to given timestamp based on current block time
-export function setTimeTo(timestamp) {
+export async function setTimeTo(timestamp) {
```

Comment:
This function does not use `await`, so doesn't need `async`. This whole file could use a good rewrite, starting with a `toPromise` wrapper function for `web3` functions.